src/utils/constants.ts

```diff
@@ -136,3 +136,8 @@ export const MAX_LEGS_LENGTH = 10;
  * Default CDD ID associated with an Identity on chain. Used for Identities onboarded without PUIS
  */
 export const DEFAULT_CDD_ID = '0x0000000000000000000000000000000000000000000000000000000000000000';
+
+/**
+ * Minimum version of Middleware V2 GraphQL Service (SubQuery) that is compatible with this version of the SDK
+ */
+export const MINIMUM_SQ_VERSION = '9.7.0-alpha.2';
```

Comment:
Just want to make sure, does the semver library work with "alpha" tag, as in if we had SQ version `9.8.0`, would things still be OK.

Reply:
Yes, this works correctly